docs/wiki/boards/PIKOBLX.md

```diff
@@ -1,3 +1,3 @@
 #### This board target has been renamed to FF_PIKOBLX.
 
-[FF_PIKOBLK board wiki page](Board---FF_PIKOBLX)
+[FF_PIKOBLK board wiki page](FF_PIKOBLX)
```

Comment:
```suggestion
[FF_PIKOBLK board wiki page](docs/wiki/boards/FF_PIKOBLX)
```

Reply:
No, this is how a link must be formatted when the file is at the same directory level.  Otherwise the link will fail when built.